nixos/modules/services/monitoring/kthxbye.nix

```diff
@@ -0,0 +1,159 @@
+{ config, pkgs, lib, ... }:
+with lib;
+
+let
+  cfg = config.services.kthxbye;
+in
+
+{
+  options.services.kthxbye = {
+    enable = mkEnableOption (mdDoc "kthxbye alert acknowledgement management daemon");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.kthxbye;
+      defaultText = literalExpression "pkgs.kthxbye";
+      description = mdDoc ''
+        The kthxbye package that should be used.
+      '';
+    };
+
+    openFirewall = mkOption {
+      type = types.bool;
+      default = false;
+      description = mdDoc ''
+        Open ports needed for the daemon to function.
+      '';
+      example = true;
+    };
+
+    extraOptions = mkOption {
+      type = with types; listOf str;
+      default = [];
+      description = mdDoc ''
+        Extra command line options.
```

Comment:
What options are available? Please link to upstream documentation.

Reply:
Since the author didn't document many of the available options, I had to find them by running `kthxbye --help`. Some of the options can be found in the upstream repo's README, but [the main go module](https://github.com/prymitive/kthxbye/blob/fd7f90469088b50f890e7692537d3ae6fa7973c0/cmd/kthxbye/main.go#L53) contains all of the available option, with a short description.

Reply:
Might be worth just linking to the README then... Unfortunate that they weren't better documented.